packages/core/src/components/TextField/Styled/TextField.styled.tsx

```diff
@@ -22,22 +33,34 @@ const styleWithLabel = ({ variant }: StyledProps) => {
     `;
 };
 
+const setFontStyle = ({ theme, size, multiline }: StyledProps) => {
+    if (multiline) {
+        return getFontStyle({ theme, fontVariant: 'body2' });
+    }
+    return getFontStyle({ theme, fontVariant: theme.textField.textVariant[size] });
+};
+
+const setHeight = ({ multiline }: StyledProps) => multiline && `height: 100%`;
+
 export const Input = styled('input')<StyledProps>`
     color: ${({ variant, theme }) => theme.textField[variant].default.textColor};
     width: 100%;
+    ${setHeight};
```

Comment:
You can write this directly over here only.
height: ${({multiline}) => multiline && '100%'}